Nos/Views/Home/HomeFeedView.swift

```diff
@@ -63,25 +67,33 @@ struct HomeFeedView: View {
 
     var body: some View {
         ZStack {
-            PagedNoteListView(
-                refreshController: $refreshController,
-                databaseFilter: homeFeedFetchRequest,
-                relayFilter: homeFeedFilter,
-                relay: selectedRelay,
-                managedObjectContext: viewContext,
-                tab: .home,
-                header: {
-                    EmptyView()
-                },
-                emptyPlaceholder: {
-                    VStack {
-                        Text(.localizable.noEvents)
-                            .padding()
+            VStack(spacing: 8) {
+                TipView(welcomeTip)
```

Comment:
Can you add `.readabilityPadding()` to the tip view?

Reply:
done!